lib/src/date-input/Calendar.tsx

```diff
@@ -0,0 +1,495 @@
+              onClick={() => setInnerDate(innerDate.set("month", innerDate.get("month") - 1))}
+            >
+              {leftCaret}
+            </PickerHeaderButton>
+            <HeaderMonthYear aria-live="polite">{innerDate.format("MMMM YYYY")}</HeaderMonthYear>
+            <PickerHeaderButton
+              aria-label="next month"
+              onClick={() => setInnerDate(innerDate.set("month", innerDate.get("month") + 1))}
+            >
+              {rightCaret}
+            </PickerHeaderButton>
+          </PickerHeader>
+          <WeekHeader>
+            {weekDays.map((weekDay) => (
+              <DayHeaderCell key={weekDay}>{weekDay}</DayHeaderCell>
+            ))}
+          </WeekHeader>
+
+          <DayCellsContainer>
+            {dayCells.map((day) =>
+              day !== 0 ? (
+                <DayCell
+                  onKeyDown={(event) => handleDayKeyboardEvent(event, day)}
+                  id={`day_${day}`}
+                  aria-label={day}
+                  onClick={() => handleDateSelect(day, "date")}
+                  selected={day === innerDate.get("date") && selectedDate.isSame(innerDate, "day")}
+                  autoFocus={day === innerDate.get("date") && selectedDate.isSame(innerDate, "day")}
+                  aria-selected={day === innerDate.get("date") && selectedDate.isSame(innerDate, "day")}
+                >
+                  {day}
+                </DayCell>
+              ) : (
+                <EmptyDayCell />
+              )
+            )}
+          </DayCellsContainer>
+        </Calendar>
+      )}
+      {content === "yearPicker" && (
+        <YearPicker>
+          {yearList.map((year) => (
+            <YearPickerButton
+              aria-label={year}
+              selected={selectedDate.get("year") === year}
+              autoFocus={selectedDate.get("year") === year}
+              onClick={() => {
+                handleDateSelect(year, "year");
+                setContent("calendar");
+              }}
+            >
+              <YearPickerButtonLabel selected={selectedDate.get("year") === year}>{year}</YearPickerButtonLabel>
+            </YearPickerButton>
+          ))}
+        </YearPicker>
+      )}
+      {content === "monthPicker" && (
+        <MonthPicker>
+          {months.map((month, index) => (
+            <MonthPickerButton
+              aria-label={month}
+              selected={selectedDate.get("month") === index}
+              autoFocus={selectedDate.get("month") === index}
+              onClick={() => {
+                handleDateSelect(index, "month");
+                setContent("calendar");
+              }}
+            >
+              {month}
+            </MonthPickerButton>
+          ))}
+        </MonthPicker>
+      )}
+    </DatePicker>
+  );
+};
+
+const DatePicker = styled.div`
+  box-shadow: 0px 5px 5px -3px rgb(0 0 0 / 20%), 0px 8px 10px 1px rgb(0 0 0 / 14%), 0px 3px 14px 2px rgb(0 0 0 / 12%);
```

Comment:
I think we should use the `box-shadow` of the other components with a dialog associated, such as the Dropdown or the Select, for consistency. Maybe Raffy can give as and advice.

Reply:
After consulting with Raffy he agrees that for consistency we should have the same `box-shadow`